installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -44,6 +44,14 @@ vagrant_dev_box_add() {
     fi
 }
 
+vagrant_dev_enable_mounts() {
+    if [[ ${vagrant_dev_no_mounts:+1} ]]; then
+        return
+    fi
+    perl -pi -e 's/(synced_folder.*nfs.*disabled: )true(.*)/$1false$2/' Vagrantfile
```

Comment:
vagrant_dev_vagrantfile is parameterized. If $first is set, disable mounts. This currently only done for vbguest, but now it needs to be done for this case, too. Don't edit a file that's being generated by the same script, iow.

Reply:
Can you explain the code a bit more? I understand that I can call `vagrant_dev_vagrantfile` again instead of editing it as Im doing here. I'll make that change. What I don't understand is what `$first` has to do with the mounts? From my reading of the code `$first` is gated by `vagrant_dev_no_vbguest` which I don't believe has any relationship with nfs mounts. What am I not seeing?

Reply:
$first means "first time through do something different". This is necessary for vbguest and now it is important for this case. The logic might look something like:

```sh
    if [[ ! ${vagrant_dev_no_vbguest:+1} ||  ! ${vagrant_dev_no_mounts:+1} && $has_network_manager ]]; then
        vagrant_dev_vagrantfile "$os" "$host" "$ip" '1'
        vagrant up
        if [[ ${vagrant_dev_no_vbguest:+1} ]]; then
            vagrant ssh <<'EOF'
sudo yum install -q -y kernel kernel-devel kernel-headers kernel-tools perl
EOF
        fi
        vagrant halt
    fi
```

It might be argued that this code and the subsequent few lines might be moved to `vagrant_dev_up()`, a new function.

There's an important point here on the UI. If vbguest and mounts are supplied (the default mode we would actually like to have), then there would be three boots instead of one with the existing code, and the above code does the same thing with two boots.

Reply:
Thanks. My myopia couldn't see past that I could still check vagrant_dev_* variables along with $first.

6c32e5c